### DISCUSSION TITLE
How to disable the appearance of destroyed villages in the world?

### DISCUSSION THREAD

**Discussion by VitonXL** [ldtteam/minecolonies#10091] [Q&A] [answered]
I don't understand how to remove the appearance of villages in the world.. They just appear everywhere.. And especially in vanilla villages.

**Comment by Thodor12** [comment, marked as answer]
You can use this datapack to disable worldgen colonies

https://www.curseforge.com/minecraft/texture-packs/stop-minecolonies-spawning-abandoned-colonies
